### PR TITLE
FormFeedback: remove duplicate error messages

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes https://github.com/inveniosoftware/invenio-app-rdm/issues/964

screencapture-127-0-0-1-5000-uploads-s02s0-xc746-2021-07-20-11_40_43

### CODE REVIEW

File: src/lib/components/FormFeedback.js [hunk 1]
```diff
@@ -57,12 +57,13 @@ class DisconnectedFormFeedback extends Component {
    * @returns String or React node
    */
   renderErrorMessages(messages) {
-    if (messages.length === 1) {
+    let uniqueMessages = messages.filter((v, i, a) => a.indexOf(v) === i);
```
Comment (fenekku) [inline src/lib/components/FormFeedback.js:60]: Cool, I was thinking [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) but that works :+1: .

**Review by fenekku — APPROVED** [review]
I think I am fine with the non-repeating error messages (although it could be argued both ways: clearer because showed 1 or less clear because not reflecting multiple errors).

**Action**: We will have the same issue with `Additional Descriptions`, so let's deal with it now.

Overall, I am happy the changes were pretty localized.

File: src/lib/components/FormFeedback.js (hunk 1 above)
Comment (zzacharo) [reply src/lib/components/FormFeedback.js:60]: @jrcastro2 I would go for either Set as @fenekku suggested or at least https://lodash.com/docs/4.17.15#uniqBy . It's much clearer I think the intention of what we try to achieve here.....

Comment (jrcastro2) [reply src/lib/components/FormFeedback.js]: Yes, changed to Set, thanks!

P.S: `Additional Descriptions` added.

**Comment by fenekku** [comment]
That still works for me :+1:

**Pull request merged**